dataeng/jobs/analytics/SnowflakeValidateStitch.groovy

```diff
@@ -23,12 +23,10 @@ class SnowflakeValidateStitch {
                 stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analytics tools repo to use.')
                 stringParam('APP_NAME', '', 'Application name of tables to validate.')
                 stringParam('SQOOP_START_TIME', '', 'Application name of tables to validate.')
-            }
-            environmentVariables {
-                env('KEY_PATH', allVars.get('KEY_PATH'))
-                env('PASSPHRASE_PATH', allVars.get('PASSPHRASE_PATH'))
-                env('USER', allVars.get('USER'))
-                env('ACCOUNT', allVars.get('ACCOUNT'))
+                stringParam('SNOWFLAKE_USER', 'SNOWFLAKE_TASK_AUTOMATION_USER')
+                stringParam('SNOWFLAKE_ACCOUNT', 'edx.us-east-1')
+                stringParam('SNOWFLAKE_KEY_PATH', 'snowflake/rsa_key_snowflake_task_automation_user.p8')
+                stringParam('SNOWFLAKE_PASSPHRASE_PATH', 'snowflake/rsa_key_passphrase_snowflake_task_automation_user')
```

Comment:
I have no idea why I overlooked these.  Converting them all from environment variables to job parameters so that we can configure them if necessary.